src/status_im2/common/confirmation_drawer/view.cljs

```diff
@@ -33,10 +33,14 @@
       (let [{:keys [group-chat chat-id public-key color
                     profile-picture name]} context
             id                             (or chat-id public-key)
-            display-name                   (or
-                                            (when-not group-chat
-                                              (rf/sub [:contacts/contact-name-by-identity id]))
-                                            name)
+            contact-name-by-identity       (when-not group-chat
+                                             (rf/sub [:contacts/contact-name-by-identity id]))
+            display-name                   (cond
+                                             (= contact-name-by-identity
```

Comment:
is this common logic that is reused on other screens? Perhaps it's better in its own sub ?

Reply:
I think `group-chat` and `name` vars comes from the context var, I'll see if I can make it more reusable.